client/README.md

```diff
@@ -0,0 +1,233 @@
+Here's a **refined and corrected** version of your `README.md` file with improved clarity, grammar, and formatting:
+
+---
+
+# **Resource Sharing Client**
+
+This package provides a **ResourceSharing client** that resource plugins can use to **implement access control** by communicating with the **OpenSearch Security Plugin**.
```

Comment:
How do plugins consume this? Is this published as a separate maven package?

This README should probably have instructions on how to consume it.